### PR TITLE
fixed duplicate line breaks in email messages

### PR DESCRIPTION
**Pull request by deki**
similar to the fix in #606

### CODE REVIEW

File: adm_program/system/classes/email.php [hunk 1]
```diff
@@ -280,11 +280,7 @@ public function setText($text)
     {
         // Erst mal die Zeilenumbrueche innerhalb des Mailtextes umwandeln in einfache Umbrueche
         // statt \r und \r\n nur noch \n
-        $replaceArray = array(
-            "\r"   => "\n",
```
Comment (ximex) [inline adm_program/system/classes/email.php]: @deki could you test it if it also works if you change line 284 and 285? I think this only is the wrong order

Comment (deki) [reply adm_program/system/classes/email.php]: You are right but shouldn't we use the same fix for all occurences? That's why I adapted the fix that was applied to systemmail.php in #606.

Comment (Fasse) [reply adm_program/system/classes/email.php]: I also don't understand why we must replace `\r`. 
I know `\n` for Linux and `\r\n` for Windows.

Comment (ximex) [reply adm_program/system/classes/email.php]: hmmm... right. why have we done this?

**Comment by ximex** [comment]
@deki could you also change the comment?

**Comment by deki** [comment]
done

**Comment by Fasse** [comment]
Thanks for the fix

**Pull request merged**